### DISCUSSION TITLE
KinkMen matching help - GEVI?? Other Site?

### DISCUSSION THREAD

**Discussion by jocklover** [CodyBerenson/PGMA-Modernized#178] [Q&A]
I am trying to match a bunch of KinkMen scenes and am striking out.
I see that IAFD does list the scenes and am using it in the meantime but obs not getting posters.

I found GEVI has a good listing of KinkMen scenes. However I am running into lots of issues with PGMA match failures. This appears to not be a failure of PGMA as **I cannot even get GEVI to pull up the titles by search** - I am having to back into them using Actor to find them. Tips/Tricks/Suggestions??? 
A sample of title issues:
(Kinky Bites) - Greedy Boy- Zak Bishop Fucked RAW By Daddy Trent Summers (2021)
https://gayeroticvideoindex.com/episode/159338
(Bound Gods) - A Date With Mr Wilde (2016)
https://gayeroticvideoindex.com/episode/146694
(Bound Gods) - Cock Worship (2010)
https://gayeroticvideoindex.com/episode/146992
(I do have a handful of successful matches occurring.

I have peaked in a bunch of supported sites and not had good luck finding KinkMen listings - any suggestions for sites with comprehensive listings?

**Comment by JPH71** [comment]
This should be sorted by the New GEVIScenes agent....
Cody has provided instructions...
